scripts/docker_build_push.sh

```diff
@@ -70,6 +94,22 @@ DOCKER_BUILDKIT=1 docker build --target lean \
   --label "build_actor=${GITHUB_ACTOR}" \
   .
 
+
+#
+# Build the "lean39" image
+#
+DOCKER_BUILDKIT=1 docker build --target lean \
+  -t "${REPO_NAME}:${SHA}-py39" \
+  -t "${REPO_NAME}:${REFSPEC}-py39" \
+  -t "${REPO_NAME}:${LATEST_TAG}-py39" \
+  --build-arg PY_VER="3.9-slim-bullseye"\
+  --label "sha=${SHA}" \
+  --label "built_at=$(date)" \
+  --label "target=lean39" \
+  --label "build_actor=${GITHUB_ACTOR}" \
+  .
```

Comment:
Why a new image based on a different base image (`bookworm` vs `bullseye`)? I think instead of creating a new `lean39` image based on `bullseye`, you could just add the tag to the already existing `lean` image.

Reply:
We also received feedback from the community that they would like to see an image based on bookworm, so we're considering supporting a few versions of Ubuntu instead of just one.

Reply:
understood, thanks!